addons/test_mail/tests/test_performance.py

```diff
@@ -220,7 +220,7 @@ def test_message_assignation_email(self):
     @warmup
     def test_message_assignation_inbox(self):
         record = self.env['mail.test.track'].create({'name': 'Test'})
-        with self.assertQueryCount(__system__=34, emp=40):  # test_mail only: 34 - 40
+        with self.assertQueryCount(__system__=39, emp=46):  # test_mail only: 36 - 46
```

Comment:
I don't understand why we have so much query count added. Do you know why or should I take a deeper look?